Source/Core/Common/Analytics.cpp

```diff
@@ -76,15 +76,15 @@ AnalyticsReportBuilder::AnalyticsReportBuilder()
   m_report.push_back(WIRE_FORMAT_VERSION);
 }
 
-void AnalyticsReportBuilder::AppendSerializedValue(std::string* report, const std::string& v)
+void AnalyticsReportBuilder::AppendSerializedValue(std::string* report, std::string_view v)
 {
   AppendType(report, TypeId::STRING);
   AppendBytes(report, reinterpret_cast<const u8*>(v.data()), static_cast<u32>(v.size()));
 }
 
 void AnalyticsReportBuilder::AppendSerializedValue(std::string* report, const char* v)
```

Comment:
I believe it would be a good idea to leave this note from the commit message here as comment (the fact that it would call the `bool` overload with a `char*` instead of the `std::string_view` one); because it seems too easy to overlook when someone else is going over the code, potentially attempting to simplify.

Reply:
Good idea. I've updated the commit to include it.